client/modules/IDE/components/CollectionList/CollectionList.jsx

```diff
@@ -1,313 +1,206 @@
+/* eslint-disable react/react-in-jsx-scope */
```

Comment:
You should import `React` rather than disabling the warning.  `import React, { useState, useEffect } from 'react';`